packages/icons/config/fabric-icons-0.json

```diff
@@ -120,10 +120,6 @@
 			"name": "Lock",
 			"unicode": "E72E"
```

Comment:
This isn't the process in which we update the icon partitions. @lynamemi and @battletoilet can help work with you on getting the root partition updated. I would like it to be automated, but it's not currently.